mitosheet/mitosheet/mito_widget.py

```diff
@@ -34,6 +35,11 @@
                                     UJ_RECEIVED_TOURS, UJ_USER_EMAIL)
 from mitosheet.user.utils import is_excel_import_enabled, is_pro, is_running_test
 
+try:
+    from mitosheet_helper_config import MITO_ENTERPRISE_CONFIGURATION 
+except ImportError:
+    MITO_ENTERPRISE_CONFIGURATION = None
```

Comment:
Another potentially sensible option here is to have a default config ourself. 

FWIW, I think we should be _really thoughtful_ about what we add to this config. We're gonna get an explosion of code paths if we're casing on random stuff throughout the codebase, which is not gonna be fun to manage and test.